src/axolotl/cli/__init__.py

```diff
@@ -466,7 +466,7 @@ def load_datasets(
         cli_args.debug
         or cfg.debug
         or cli_args.debug_text_only
-        or cli_args.debug_num_examples
+        or int(cli_args.debug_num_examples) > 0
```

Comment:
```suggestion
        or (cli_args.debug_num_examples and int(cli_args.debug_num_examples) > 0)
```

Do you perhaps need to adjust to check for None/str?

Reply:
it will always get cast to an int. ` debug_num_examples: int = field(default=0)`